index.html

```diff
@@ -68,7 +68,7 @@ <h2>PID Review</h2>
 
   <tr>
     <td>
-      <a href="./FilterTool" style="margin: 20px; display: inline-block;"><img src="../images/FilterTool_Icon.png" style="width:200px"></a>
+      <a href="./FilterTool" style="margin: 20px; display: inline-block;"><img src="images/FilterTool_Icon.png" style="width:200px"></a>
```

Comment:
I know it was not changed in this PR, but the description has this awkward explanation "For filter setup the FilterReview tool it is recommended it provides the same bode plot feedback." - could you clean that up?